src/track.c

```diff
@@ -58,6 +59,13 @@ char lock_detect_params_string[24] = LD_PARAMS_DISABLE;
 bool use_alias_detection = true;
 
 #define CN0_EST_LPF_CUTOFF 5
+#define GPS_WEEK_LENGTH_ms (7*24*60*60*1000)
```

Comment:
Can you use `WEEK_SECS` from `libswiftnav/time.h`?

e.g. `#define GPS_WEEK_LENGTH_ms (WEEK_SECS*1000)`